### DISCUSSION TITLE
Github Personal Access Token Expirty color scheme is not good

### DISCUSSION THREAD

**Discussion by ajeeb-siterx** [community/community#20499] [Accessibility]
On this page, both expired and about to expired token message color is same.
https://github.com/settings/tokens

Before Expiry:
 Expires on Fri, Jul 15 2022. 
After Expiry:
 Expired on Fri, Jul 15 2022. 
 
 Expected:
 Both text should be in different color.
 Actual:
 Both color is the same, yellow.

**Comment by y9su** [comment]
This could possible be a bug, you should report it to- https://support.github.com/contact/bug-report
Please accept my answer if I helped!

**Comment by cehfisher** [comment]
Hello @ajeeb-siterx - thank you for providing this feedback! We wanted to follow up on this reported issue.

When we tested this issue using the beta version of the [fine-grained personal access tokens](https://docs.github.com/en/authentication/keeping-your-account-and-data-secure/managing-your-personal-access-tokens#fine-grained-personal-access-tokens), we saw something different than what was reported. While the text was the same color for the active and expired tokens, the expired token included a yellow caution icon and link which read "This token has expired."

Is this different from what you are seeing? If this issue is now fixed, could you please let us know or close out this issue? If it is not fixed, what additional changes are needed? Please include screenshots, if possible. Thank you!

<img width="781" alt="Screenshot in dark default mode showing the visual explained above." src="https://github.com/community/community/assets/15240149/b75c9c0a-e0dc-4ef0-87f3-087c915b1f90">

<img width="788" alt="Screenshot in light default mode showing the visual explained above." src="https://github.com/community/community/assets/15240149/4b4e1db6-9bb6-43a3-aac8-b87f08d687f9">
